src/main/resources/computercraft.accesswidener

```diff
@@ -20,3 +20,4 @@ accessible field net/minecraft/client/render/model/BakedQuad sprite Lnet/minecra
 accessible field net/minecraft/client/util/math/AffineTransformation translation Lnet/minecraft/client/util/math/Vector3f;
 accessible field net/minecraft/client/util/math/AffineTransformation scale Lnet/minecraft/client/util/math/Vector3f;
 accessible field net/minecraft/client/util/math/AffineTransformation rotation1 Lnet/minecraft/util/math/Quaternion;
+accessible field net/minecraft/block/entity/SignBlockEntity text [Lnet/minecraft/text/Text;
```

Comment:
Accessor mixin do less brr

Reply:
blame zundrel, i'm following the convention

Reply:
accessor mixin many more line of code, more line of code = more file size. we are saving size here :crab:.

Reply:
i wil comit aw delet

Reply:
@i509VCB I have avenged you https://github.com/Zundrel/cc-tweaked-fabric/commit/1346a26179b9444aea4f1e5c6aff423370b5a366